draft-ietf-rats-reference-interaction-models.md

```diff
@@ -136,99 +136,96 @@ The Verifier's assessment in the form of Attestation Results is created based on
 
 # Essential Requirements
 
-In order to ensure appropriate conveyance of Evidence, there exist essential requirements which MUST be fulfilled:
+In order to ensure appropriate conveyance of Evidence, the following requirements MUST be fulfilled:
 
 Integrity:
-
-: Information provided by an Attester MUST be integral. This may be achieved by means of a digital signature over Attestation Evidence. The signature may be symmetric, such as an HMAC, or asymmetric, such as ECDSA.
+: Information provided by an Attester MUST NOT have been altered since it was created. This may be achieved through a digital signature over Attestation Evidencewhich may be symmetric, like an HMAC, or asymmetric, like ECDSA.
 
 Authentication:
-
-: The information provided by the Attester MUST be authentic. For that purpose, the Attester should authenticate itself to the Verifier. This may be an implicit authentication by means of a digital signature over the Attestation Evidence, which does not require additional protocol steps, or may be achieved by using a confidential channel by means of encryption.
+: The information provided by the Attester MUST be authentic. To do this, the Attester should authenticate itself to the Verifier. This can be done through implicit authentication using a digital signature over the Attestation Evidence, which does not require additional protocol steps, or by using a confidential channel with encryption.
 
 ## Endorsement of Attesting Environments
 
-Via its Attesting Environments, an Attester only generates Evidence about its Target Environments.
-After being appraised to be trustworthy, a Target Environment may become a new Attesting Environment in charge of generating Evidence for further Target Environments.
-{{-RATS}} explains this as Layered Attestation.
-Layered Attestation has to start with an initial Attesting Environment. In essence, there cannot be turtles all the way down {{turtles}}.
-At this rock bottom of Layered Attestation, the Attesting Environments are always called Roots of Trust (RoT).
-An Attester cannot generate Evidence about its own RoTs by design.
-As a consequence, a Verifier requires trustable statements about this subset of Attesting Environments from a different source than the Attester itself.
-The corresponding trustable statements are called Endorsements and originate from external, trustable entities that take on the role of an Endorser (e.g., supply chain entities).
+An Attester only generates Evidence about its Target Environments through its Attesting Environments.
+Once a Target Environment is appraised as trustworthy, it may become a new Attesting Environment responsible for generating Evidence for other Target Environments.
+This is known as Layered Attestation, as explained in {{Section 3.2 of -RATS}}.
+
+Since there can't be an infinite chain of Attesting Environments {{{turtles}}, Layered Attestation has to start with an initial Attesting Environment.
```

Comment:
```suggestion
Since there can't be an infinite chain of Attesting Environments {{turtles}}, Layered Attestation has to start with an initial Attesting Environment.
```